proto/gaia/globalfee/v1beta1/genesis.proto

```diff
@@ -28,4 +28,14 @@ message Params {
     (gogoproto.moretags) = "yaml:\"minimum_gas_prices\"",
     (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.DecCoins"
   ];
+  // bypass_min_fee_msg_types defines a list of IBC message type urls
+  // that are free of fee charge
+  repeated string bypass_min_fee_msg_types = 2 [
+    (gogoproto.jsontag) = "bypass_min_fee_msg_types,omitempty",
+    (gogoproto.moretags) = "yaml:\"bypass_min_fee_msg_types\""
+  ];
+
+  // max_total_bypass_min_fee_msg_gas_usage defines the total maximum gas usage
```

Comment:
Can you expand on this? Something like "defines the total maximum gas usage allowed for a transaction containing only messages of type bypass_min_fee_msg_types to bypass fee charge".